packages/components/legacy_table/src/legacy_table.tsx

```diff
@@ -0,0 +1,101 @@
+import { BoxProps } from "@buoysoftware/anchor-layout";
+import { CellProps, Table, TableCellConfig, TableProps } from "@buoysoftware/anchor-table";
+import { LegacyTextVariant } from "@buoysoftware/anchor-ui-legacy-text";
+import { TFunction } from "i18next";
+import snakeCase from "lodash/snakeCase";
+import { useTranslation } from "react-i18next";
+
+type LegacyTextProps = BoxProps & { variant?: LegacyTextVariant };
+type LegacyCellProps = Partial<LegacyTextProps> | undefined;
+
+type CellT = TFunction<string[], undefined, string[]>;
+
+type LegacyTableCellConfig<RecordData> = Omit<TableCellConfig<RecordData>, 'cellProps' | 'render'> & {
+  cellProps?: (rowData: RecordData) => LegacyCellProps;
+  render?: (
+    rowData: RecordData,
+    tableT: TFunction,
+    cellT: (...params: Parameters<CellT>) => string
+  ) => React.ReactNode;
+  translationNamespace?: string;
+  translationKeyPrefix?: string;
```

Comment:
We should also add backward compatibly for the `t` prop. `t` was deprecated in favor of `translationNamespace` and `translationKeyPrefix` but its almost definitely still in use in BuoyRails.

Reply:
Hey @acostanzo, I took a look in both Wharf and BuoyRails for usage of `t`, and I couldn't find any 🤔 

1. Did I miss it somewhere in those or is there another repository that could potentially use the `TableBuilder` and the `t` prop?
2. I'm unclear what I would do with the `t` prop. In the original `TableBuilder`, it would pass that prop down to `TCell` and there it would leverage. In the new design system, the `Td` component doesn't seem to use it. LMK what your suggestions are. Thanks! 🙏

Reply:
They are definitely few and far between as we had mostly moved to translationNamespace & translationKeyPrefix. You can see examples in some of the event tables in BuoyRails. For example https://github.com/BuoySoftware/BuoyRails/blob/main/app/javascript/apps/buoy/components/Operations/DonorEventsTable/DonorEventDetailsModal/AdverseReactionEvent.tsx#L41

So I think the t option would needed to be added to the legacy cell props. The old ranking is

1. t passed to cell props
2. t made up of translationNamespace & translationKeyPrefix
3. tableT

Reply:
I realize this bit is a little bonkers lol, let me know if you want to hop on a call to discuss further

Reply:
Thank you for that example! This makes sense now. Updated the component to allow receiving `t` 🙌